apps/demos/Demos/Chat/AIAndChatbotIntegration/Angular/app/app.component.ts

```diff
@@ -55,6 +58,21 @@ export class AppComponent {
     this.regenerationText = this.appService.REGENERATION_TEXT;
     this.copyButtonIcon = 'copy';
     this.isDisabled = false;
+
+    const provider: AIProvider = {
+      sendRequest(params) {
+        return {
+          abort: () => {},
+          promise: Promise.resolve('AI response'),
+        }
+      },
+    };
+    const ai: AIClass = new AI(provider);
+
+    ai.translate(
+      { text: 'text', lang: 'lang' },
+      { onComplete: (result) => { console.log(result) } },
+    );
```

Comment:
What is this for? Looks like a spike you forgot to remove

Reply:
[6fc0a500f05c7afd090d61a12403b7d8ea7ea0b4]